### PR TITLE
[LTE][AGW] NSA add UE Additional security capabilities in NAS SMC

### LINKED ISSUE #5811: [NSA] Add "Replayed UE Additional Security Capability" IE in Security Mode Command

### PR DESCRIPTION
**Pull request by lionelgo**
## Summary

Add UE Additional security capabilities in NAS SECURITY_MODE_COMMAND.
Add NRUESecurityCapabilities in InitialContextSetupRequest.
Assuming MME always acknowledge NSA for incoming en-dc featured UEs
Changes made to be compliant with 3GPP specs

## Test Plan

Tested with ds-tester and OAI CUPS
See ds-tester scenario file and PCAP

## Additional Information

- [ ] This change is backwards-breaking
[PR5811.zip](https://github.com/magma/magma/files/6609189/PR5811.zip)

### CODE REVIEW

File: lte/gateway/c/core/oai/tasks/nas/emm/Attach.c [hunk 1]
```diff
@@ -2420,6 +2425,43 @@ static bool emm_attach_ies_have_changed(
...
+        ue_id);
+    OAILOG_FUNC_RETURN(LOG_NAS_EMM, true);
+  }
+
+  if ((ies1->ueadditionalsecuritycapability) &&
+      (ies2->ueadditionalsecuritycapability)) {
+    if (memcmp(
+            ies1->ueadditionalsecuritycapability,
+            ies2->ueadditionalsecuritycapability,
+            sizeof(*ies2->ueadditionalsecuritycapability)) != 0) {
```
Comment (ulaskozat) [inline lte/gateway/c/core/oai/tasks/nas/emm/Attach.c]: can you use brackets when both * and -> used to make it more readable?

**Review by ulaskozat — APPROVED** [review]
LGTM with nit comment.

**Comment by ulaskozat** [comment]
can you also run integ_test locally and note that it is passing before merging?

**Comment by ulaskozat** [comment]
@lionelgo you need to generate go proto files by running `build.py -g` under `orc8r/cloud/docker/` and add them to your PR.

**Review by ardzoht — APPROVED** [review]
lgtm, as @ulaskozat pointed out please run `./build.py -g` on docker path to generate Golang protos and commit your changes

**Pull request merged**